### ISSUE TITLE
"File in use" error when attempting to `releasify` with windows defender enabled

### ISSUE THREAD

**Issue by peppy** [clowd/Clowd.Squirrel#88]
When attempting to do a build from `%userprofile%\Desktop` (not sure if relevant), builds will 100% fail with window defender enabled:

```csharp
27792   Running squirrel build...
28722   Running C:\Users\Dean\.nuget\packages\clowd.squirrel\2.9.40\tools\Squirrel.exe releasify --package=C:\Users\Dean\Desktop\osu-deploy\staging\osulazer.12.3.4.nupkg --releaseDir=C:\Users\Dean\Desktop\osu-deploy\releases --icon=C:\Users\Dean\Desktop\osu\osu.Desktop\lazer.ico --appIcon=C:\Users\Dean\Desktop\osu\osu.Desktop\lazer.ico --splashImage=C:\Users\Dean\Desktop\osu\assets\lazer-nuget.png ...
39154   [INFO] DotnetUtil: Extracting Update.exe resources to temp directory
[INFO] DotnetUtil: Patching Update.exe icon
[INFO] DotnetUtil: Re-packing Update.exe bundle
[INFO] Program: Creating release for package: C:\Users\Dean\Desktop\osu-deploy\releases\osulazer.12.3.4.nupkg
[INFO] ReleasePackage: Creating release package: C:\Users\Dean\Desktop\osu-deploy\releases\osulazer.12.3.4.nupkg => C:\Users\Dean\Desktop\osu-deploy\releases\osulazer-12.3.4-full.nupkg
[INFO] ReleasePackage: Removing unnecessary data
[INFO] Program: There are 1 SquirrelAwareApp's. Binaries will be executed during install/update/uninstall hooks.
[INFO] Program:   Detected SquirrelAwareApp 'osu!.exe' (arch: x64)
[INFO] Program: Package Architecture (detected from SquirrelAwareApp's): x64
[INFO] Program: Creating 2 stub executables
[INFO] Program: Using app icon from command line arguments
[INFO] EasyZip: Compressing '%localappdata%\SquirrelClowdTemp\tempe' to 'C:\Users\Dean\Desktop\osu-deploy\releases\osulazer-12.3.4-full.nupkg' using 7z...
[INFO] Program: Creating Setup bundle

[ERRO] System.IO.IOException: The process cannot access the file 'C:\Users\Dean\Desktop\osu-deploy\releases\osulazerSetup.exe' because it is being used by another process.
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String , FileMode , FileAccess , FileShare , FileOptions )
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String , FileMode , FileAccess , FileShare , FileOptions , Int64 )
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String , FileMode , FileAccess , FileShare , FileOptions , Int64 )
   at System.IO.File.Open(String , FileMode , FileAccess , FileShare )
   at System.IO.File.Open(String , FileMode , FileAccess )
   at Squirrel.Shared.SetupBundle.CreatePackageBundle(String setupPath, String packagePath)
   at SquirrelCli.Program.Releasify(ReleasifyOptions options) in ./Program.cs:line 378
   at SquirrelCli.CommandAction`1.Execute(IEnumerable`1 args) in ./ValidatedOptionSet.cs:line 168
   at SquirrelCli.Program.Main(String[] args) in ./Program.cs:line 87
```

![Parallels Desktop 2022-06-15 at 05 31 26](https://user-images.githubusercontent.com/191335/173749506-79dc23f0-367b-4362-9fa5-834e481f49d9.png)

Disabling windows defender resolves the issue, so this is low priority to us. Not sure how relevant our project is to this happening, but likely depends on the speed of execution and the size of the generated file.

**Comment by caesay** [comment]
Related to https://github.com/clowd/Clowd.Squirrel/issues/72

**Comment by caesay** [comment]
Closed by 12942118b541a67b1c29a5f2c39a0261149497fe
